preload-split-package.php

```diff
@@ -333,3 +333,4 @@
 require_once __DIR__ . '/../../../vendor/phpstan/phpdoc-parser/src/Printer/DiffElem.php';
 require_once __DIR__ . '/../../../vendor/phpstan/phpdoc-parser/src/Printer/Differ.php';
 require_once __DIR__ . '/../../../vendor/phpstan/phpdoc-parser/src/Printer/Printer.php';
+require_once __DIR__ . '/../../../vendor/nette/utils/src/Utils/Reflection.php';
```

Comment:
this will be replaced via script: `composer preload` which run weekly, so better to move append in Preload script:

https://github.com/rectorphp/rector-src/blob/3bc82de4d5b27b2852e6a1ecbc68607cf4d000ce/build/build-preload.php#L108-L118

Reply:
It was a try which didn't work. 

I need to find a way to test my change. But couldn't find one yet. 

Afterwards we can clean it up in case it works

Reply:
No worries, I can update the script if this got merged so this append not cleared on next regenerate ;)